### PR TITLE
Color Theme Setup

### PR DESCRIPTION
**Pull request by WookieeMatt**
Addresses https://github.com/WookieeMatt/Essence20/issues/203

### CODE REVIEW

File: module/settings.js [hunk 1]
```diff
@@ -82,4 +82,9 @@ export const registerSettings = function () {
     type: Number,
     config: false,
   });
+
+  /* -------------------------------------------- */
+  /*  Theme Settings                              */
+  /* -------------------------------------------- */
+
```
Comment (phildominguez) [inline module/settings.js:89]: Can remove this

File: templates/actor/parts/headers/actor-pr-header.hbs [hunk 2]
```diff
@@ -16,5 +16,9 @@
   <label for="system.level" class="resource-label">{{localize 'E20.ActorLevel'}}</label>
   <input class="centered-input" style="width: 100%;" type="number" name="system.level" value="{{system.level}}" />
 </div>
+<div class="pr-header-color">
+  <label for="system.color" class="resource-label">{{localize 'E20.ActorColor'}}</label>
+  <input type="color" class="picker" name="system.color" value="{{system.color}}" />
+</div>
```
Comment (phildominguez) [inline templates/actor/parts/headers/actor-pr-header.hbs:22]: image
Could the "Color" label be centered?

File: system.json [hunk 3]
```diff
@@ -1,20 +1,22 @@
 {
```
Comment (phildominguez) [inline system.json:1]: Is this a new system.json format for v10 or something?

**Review by phildominguez — CHANGES_REQUESTED** [review]
Looks great! Just a few comments. 

I'd also be in favor of making the default background a dark grey gradient instead of the current blueish one, so it doesn't clash with whatever color they pick. Probably something neutral for the tab and skill header colors, too. Thoughts on that @guswadsworth12 ?

**Comment by guswadsworth12** [comment]
> Looks great! Just a few comments.
> 
> I'd also be in favor of making the default background a dark grey gradient instead of the current blueish one, so it doesn't clash with whatever color they pick. Probably something neutral for the tab and skill header colors, too. Thoughts on that @guswadsworth12 ?

This is probably a good idea. Let me see what I can do.

File: templates/actor/parts/headers/actor-pr-header.hbs (hunk 2 above)
Comment (guswadsworth12) [reply templates/actor/parts/headers/actor-pr-header.hbs:22]: This would need a new resource-label Class just for this one too show centered

Comment (phildominguez) [reply templates/actor/parts/headers/actor-pr-header.hbs:22]: Looks good now

File: module/settings.js (hunk 1 above)
Comment (phildominguez) [reply module/settings.js]: I've removed this

**Review by phildominguez — APPROVED** [review]

**Pull request merged**